src/M6Web/Bundle/LogBridgeBundle/Config/FilterCollection.php

```diff
@@ -90,10 +88,8 @@ public function remove(Filter $filter)
      * Get item
      *
      * @param int $iterator
-     *
-     * @return mixed
      */
-    public function get($iterator)
+    public function get($iterator): mixed
```

Comment:
```suggestion
    public function get($iterator): ?Filter
```


ℹ️ this can also be done for the `getByName()` function